run.js

```diff
@@ -325,25 +325,25 @@ async function templatePageGenerator(templateIndex, templatePath, template) {
 
     const requestTypes = {};
     for(let type of template.getRequestTypes()) {
-        const classDecl = template.getModelManager().getType(type);
-        const sampleInstance = template.getFactory().newResource( classDecl.getNamespace(), classDecl.getName(), uuidv1(), sampleGenerationOptions);
-        requestTypes[type] = JSON.stringify(sampleInstance, null, 4);
+        requestTypes[type] = generateSampleJSON(type);
     }
 
     const responseTypes = {};
     for(let type of template.getResponseTypes()) {
-        const classDecl = template.getModelManager().getType(type);
-        const sampleInstance = template.getFactory().newResource( classDecl.getNamespace(), classDecl.getName(), uuidv1(), sampleGenerationOptions);
-        responseTypes[type] = JSON.stringify(sampleInstance, null, 4);
+        responseTypes[type] = generateSampleJSON(type);
+    }
+
+    const stateTypes = {}
+    for(let type of template.getStateTypes()) {
+        if(type){
```

Comment:
Only required because of https://github.com/accordproject/cicero/issues/147

Reply:
Yeah - I removed that as well.